src/EditorFeatures/CSharpTest/CodeActions/ConvertIfToSwitch/ConvertIfToSwitchTests.cs

```diff
@@ -854,7 +881,7 @@ public async Task TestTrivia2()
             await TestInRegularAndScriptAsync(
 @"class C
 {
-    int M(int i)
+    int M(int i, string[] args)
```

Comment:
why these changes?

Reply:
our analysis is not syntactical so we need valid code for correct result.